GSA/digitalgov.gov
# Create Communities Section on Home Page

## Summary

Created a `communities.html` partial to be used on both the home page of digital.gov and on the communities page. Also refactored the communities cards to use a generic `usa-card.html` partial. Updated the styling in `_communities.scss` to accommodate  communities being displayed on the home page.

### Preview

[Link to Preview →](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/jtm-create-homepage-communities/)



### Solution

#### Before
<img width="1438" alt="Screenshot 2023-07-11 at 5 09 35 PM" src="https://github.com/GSA/digitalgov.gov/assets/56370370/aa81cd80-a0a8-4eb3-9130-b1097d94245d">

#### After
<img width="1436" alt="Screenshot 2023-08-01 at 9 31 52 AM" src="https://github.com/GSA/digitalgov.gov/assets/56370370/a0d2b2a2-8f0e-4580-98f7-cf3f764aee68">


### How To Test

1. Visit the [home page of digital.gov](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/jtm-create-homepage-communities/)
2. Visit the [communities page of digital.gov](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/jtm-create-homepage-communities/communities/)
3. For both pages, verify that the featured communities display properly on all screen sizes

---

### Dev Checklist
